### ISSUE TITLE
WCAG 1.3.1,WCAG 3.3.2: Ensures every form element has a label (#textfield)

### ISSUE THREAD

**Issue by oklimanova** [oklimanova/hello-world#13]
#### Issue
`Ensures every form element has a label` ([label](https://dequeuniversity.com/rules/axe/3.5/label?application=msftAI))
#### Target application
[Search of EO 12866 Meeting](https://training.reginfo.gov/public/do/eom12866Search)
#### Element path
#textfield
#### Snippet
`<input type="text" alt="Search Content" style="height:15px; width:142px;margin-top:-2px;margin-left:30px;" name="textfield" id="textfield" class="search" value="">`
#### How to fix
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      Element has no title attribute or the title attribute is empty
#### Environment
Chrome version 84.0.4147.30
This accessibility issue was found using Accessibility Insights for Web 2.18.2 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
